locales/it/index.yml

```
@@ -2563,6 +2563,8 @@ bonus:
         title: "Vuoi davvero eliminare i seguenti metodi dal Portafoglio?"
         subtitle: "Potrai aggiungerli di nuovo in qualsiasi momento, ma dovrai reinserire i dati e ripetere le procedure di autorizzazione."
   cgn:
+    generic:
+      linkError: "Non è stato possibile aprire la url al momento, riprova"
```

Are we sure that the end user knows about what `url` is?
```suggestion
      linkError: "Non è stato possibile aprire il sito dell'operatore al momento, riprova"
```

---

and here @AliceAzzolini

---

"In questo momento, il sito dell'operatore non è raggiungibile. Riprova più tardi."
"The merchant's site can't be reached at the moment. Please, try again later."

---

```suggestion
      linkError: "In questo momento, il sito dell'operatore non è raggiungibile. Riprova più tardi."
```